Gopkg.toml

```diff
@@ -52,9 +52,10 @@ ignored = ["github.com/openshift/installer/tests*"]
   name = "sigs.k8s.io/cluster-api-provider-aws"
   source = "https://github.com/openshift/cluster-api-provider-aws.git"
   branch = "master"
+
 [[constraint]]
   name = "github.com/openshift/cluster-api-provider-libvirt"
-  branch = "master"
+  revision = "223f16433e235d5b4fce8f332abc6abcd50acaf4"
```

Comment:
Some very brief previous discussion around pinning sigs.k8s.io/cluster-api [here][1].  Personally, I prefer pinning (ideally to versioned releases that our dependencies are cutting).  But if we're transitioning to a pinned libvirt provider, maybe we should pin sigs.k8s.io/cluster-api-provider-aws and sigs.k8s.io/cluster-api as well?

[1]: https://github.com/openshift/installer/pull/573#discussion_r229898373